docs/ug/authorConfigSyntax.md

```diff
@@ -0,0 +1,107 @@
+
+| Repository's Location                                                                                                | Branch  | Author's Git Host ID | ... Hidden columns    |
+|----------------------------------------------------------------------------------------------------------------------|---------|----------------------|-----------------------|
+| https://github.com/reposense/RepoSense.git~master\|cypress\|release;https://github.com/reposense/testRepo-Alpha.git  | master  | sikai00              | --                    |
+
+<br>
+
+### Usage
+
+#### Specifying multiple `Repository's Location`
+
+To specify multiple `Repository's Location`, we can use a semicolon `;` as a separator.
+
+Below is an example:
+
+| Repository's Location                                                               | Branch  | Author's Git Host ID | ... Hidden columns    |
+|-------------------------------------------------------------------------------------|---------|----------------------|-----------------------|
+| https://github.com/reposense/RepoSense.git;https://github.com/MarkBind/markbind.git | master  | sikai00              | --                    |
+
+We have now furnished author details for both repositories in a single row, instead of having two individual rows per Repository's Location.
+
+<br>
+
+#### Specifying multiple `Repository's Location` with a single specific branch
+
+To override the Branch column with a specific branch for a particular `Repository's Location`, we can use a tilde `~` separator to denote our branch.
+
+Below is an example:
+
+| Repository's Location                                                                       | Branch  | Author's Git Host ID | ... Hidden columns    |
+|---------------------------------------------------------------------------------------------|---------|----------------------|-----------------------|
+| https://github.com/reposense/RepoSense.git~release;https://github.com/MarkBind/markbind.git | master  | sikai00              | --                    |
+
+Instead of using the branch specified in the branch column for `https://github.com/reposense/RepoSense.git`, the "release" branch will be used instead.
+
+`https://github.com/MarkBind/markbind.git` will still use the "master" branch as specified in the branch column.
+
+<br>
+
+#### Specifying multiple `Repository's Location` with multiple specific branches
+
+To list out multiple branches for a particular `Repository's Location`, we can use a pipe `|` separator to demarcate the different branches.
+
+Below is an example:
+
+| Repository's Location                                                | Branch   | Author's Git Host ID | ... Hidden columns    |
+|----------------------------------------------------------------------|----------|----------------------|-----------------------|
+| https://github.com/reposense/RepoSense.git~master\|release\|cypress; | master   | sikai00              | --                    |
```

Comment:
The table here seems to be broken:

![image](https://user-images.githubusercontent.com/50147457/229065952-d0cc28b3-64ea-4882-8b69-6c6dac101af2.png)